src/components/GetStarted/Category/index.js

```diff
@@ -28,7 +28,7 @@ const Category = ({ classes, icon: Icon, title, description, tabs, rightAlign })
     >
       <Grid item xs={12} sm={6} md={6} lg={6} className={classes.CategoryContent}>
         <div className={classes.Title}>
-          {Icon & <Icon />}
+          {Icon ? <Icon /> : null}
```

Comment:
This is not gaurd clause. it is a `conditional rendering`. But as the code is too small. it is okay..